backend/app/schemas/conversation.py

```
@@ -0,0 +1,33 @@
+from pydantic import BaseModel
+from typing import List, Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)